### ISSUE TITLE
Which Sensors, Compute are used?

### ISSUE THREAD

**Issue by drcrook1** [NVIDIA-ISAAC-ROS/isaac_ros_nvblox#100]
I'm working on modifying a radio controlled track based brush hog to mow steep slopes on my farm.  I'm trying to figure out what sensors I'll need to add.  Trying to get to PoC fairly quickly.

**Comment by UserName-wang** [comment]
Hi, interesting project!I think the most important sensors you need are a depth camera with IMU integrated and GPS sensor.在 2024年4月19日，03:30，David Crook ***@***.***> 写道：﻿
I'm working on modifying a radio controlled track based brush hog to mow steep slopes on my farm.  I'm trying to figure out what sensors I'll need to add.  Trying to get to PoC fairly quickly.

**Comment by alexmillane** [comment]
Correct. You'll need a depth camera. Realsense d455 and or the [NOVA Orin developer kit](https://robotics.segway.com/nova-dev-kit/) are good ways of getting started.
